libs/island-ui/core/src/lib/Tabs/Tabs.tsx

```diff
@@ -30,6 +31,7 @@ export const Tabs: FC<TabInterface> = ({
   tabs,
   contentBackground = 'purple100',
   size = 'md',
+  onClick: onClickHandler,
```

Comment:
This onClick event probably doesn't handle all the ways that the tabs can change, and annoying to deal with double clicks outside.

Should look into exposing an `onChange` property instead. Maybe by tracking changes to `tab.selectedId` using a ref?

Reply:
Good idea since the library does not expose listeners